dev-php/sebastian-environment/sebastian-environment-4.2.3-r1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="${PN/sebastian-//}"
+
+DESCRIPTION="Helps writing PHP code that has runtime-specific execution paths"
+HOMEPAGE="https://github.com/sebastianbergmann/environment"
+SRC_URI="https://github.com/sebastianbergmann/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sparc ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-php/fedora-autoloader
+	>=dev-lang/php-7.1:*"
+DEPEND="
+	test? (
+		${RDEPEND}
+		dev-php/phpunit )"
+
+PATCHES=(
+	"${FILESDIR}/${PV}-0001-fix-fstat-call.patch"
+	)
+
+src_prepare() {
+	default
+	if use test; then
+		mkdir "${S}"/vendor
+		cp "${FILESDIR}/${PV}"-autoload-test.php "${S}"/vendor/autoload.php || die
+	fi
+}
+
+src_install() {
+	insinto /usr/share/php/SebastianBergmann/Environment
+	doins -r src/*
+	newins "${FILESDIR}/${PV}"-autoload.php autoload.php
+	dodoc README.md
+}
+
+src_test() {
+	# phpunit --bootstrap autoload.php|| die "test suite failed"
```

Comment:
Kill this comment and add space in next line between .php and ||

Reply:
Ok done